ex03.exs

```diff
@@ -59,10 +59,52 @@ defmodule Ex03 do
       Use of language and libraries
         5 elegant use of language features or libraries
 
+
+
+  Your mission is to implement a function
+
+  pmap(collection, process_count, func)
+
+  This will take the collection, split it into n chunks, where n is
+  the process count, and then run each chunk through a regular map
+  function, but with each map running in a separate process.
   """
 
+  # Steps:
+  # 1. Chunk the collection
+  # 2. Apply function to each part of the chunk, spawn async tasks with function
+  # 3. Wait for for work
+  # 4. Concatenate it back
+
+  # API process map function that takes a list of data, # processes, and function
+  # to apply across list.
```

Comment:
I like the way you've used named functions here to make the code read more like a narrative. I don't you if you feel it, but I believe that little extra work elevates code in general, making it easier to reason about, test and change.

### +4